packages/user-group-service/src/helpers.ts

```diff
@@ -86,6 +87,9 @@ class UserGroupApiHelper {
   }
   // Helper function for rover interaction
   public roverFetch ( urlPart: String ) {
+    const httpsAgent = new https.Agent({
+      rejectUnauthorized: false,
```

Comment:
Can you remove this flag?
For local environments, we can work around this using `NODE_TLS_REJECT_UNAUTHORIZED=0`.

The other environments already have the CA cert configured, so this is not recommended.